cli/tests/pcluster/templates/test_cw_dashboard_builder.py

```diff
@@ -75,9 +75,10 @@ def _verify_ec2_metrics_conditions(cluster_config, output_yaml):
     # Check each section title
     for storage_type, storages in storage_resource.items():
         if len(storages) > 0:
-            assert_that(output_yaml).contains(storage_type_title_dict[storage_type])
+            assert_that(output_yaml).contains(storage_type_title_dict[storage_type].get("title"))
```

Comment:
nit: one way to avoid code replication is:
```python
for field in ["title", "namespace"]:
      assert_that(output_yaml).contains(storage_type_title_dict[storage_type].get(field))
```

Reply:
The reason I didn't use the for loop is that it only use for the "if" case but not in the "else" case.
Because in the "else" case, EBS and Raid both have nane space "AWS/EBS".
Change to:
```
    for storage_type, storages in storage_resource.items():
        if len(storages) > 0:
            for field in ["title", "namespace"]:
                assert_that(output_yaml).contains(storage_type_title_dict[storage_type].get(field))
        else:
            assert_that(output_yaml).does_not_contain(storage_type_title_dict[storage_type].get("title"))
```